packages/flutter_tools/test/commands.shard/hermetic/devices_test.dart

```diff
@@ -66,3 +79,12 @@ class MockProcessManager extends Mock implements ProcessManager {
     return ProcessResult(0, 0, '', '');
   }
 }
+
+class _MockDeviceManager extends DeviceManager {
+  _MockDeviceManager();
+
+  @override
+  Future<List<Device>> getAllConnectedDevices() {
+    return Future<List<Device>>.value(mockDevices.map((MockDeviceJson d) => d.dev).toList());
+  }
+}
```

Comment:
```suggestion
}

```

Reply:
Hm, interesting. I've seen complaints from the analyzer about blank lines at the end.